packages/fx-core/src/plugins/solution/fx-solution/solution.ts

```diff
@@ -376,6 +383,11 @@ export class TeamsAppSolution implements Solution {
         JSON.stringify(manifest, null, 4)
       );
       await fs.writeJSON(`${ctx.root}/permissions.json`, DEFAULT_PERMISSION_REQUEST, { spaces: 4 });
+      ctx.telemetryReporter?.sendTelemetryEvent(SolutionTelemetryEvent.Create, {
```

Comment:
You have to send telemetry before every return. And if return result is ok, add "success": "yes" to property. If return result is err, add "success": "no", and "error-type", "error-code" and "error-message" properties.